doc/run_with_mysql.md

```diff
@@ -22,8 +22,8 @@ mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0
 ```
 Using the `datasource`, you may launch an all-in-one Docker container by running:  
 ```bash
-> docker run --rm -p 8888:8888 sqlflow/sqlflow bash -c \
-"sqlflowserver --datasource='mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0' &
+> docker run --rm -p 8888:8888 -p 50051:50051 sqlflow/sqlflow bash -c \
+"sqlflowserver & SQLFLOW_DATASOURCE='mysql://root:root@tcp(127.0.0.1:3306)/iris?maxAllowedPacket=0' 
```

Comment:
Also `SQLFLOW_DATASOURCE=...` should be the same line with `jupyter`, otherwise, the `jupyter` process can't read the value of `SQLFLOW_DATASOURCE`.